dbus_next/auth.py

```diff
@@ -60,10 +60,13 @@ def _authentication_start(self) -> str:
     def _receive_line(self, line: str):
         response, args = _AuthResponse.parse(line)
 
-        if response != _AuthResponse.OK:
-            raise AuthError(f'authentication failed: {response.value}: {args}')
+        if response is _AuthResponse.OK:
+            return "NEGOTIATE_UNIX_FD"
```

Comment:
> This command may only be sent on transports that support Unix file descriptor passing.

Some people use http transport with this even though it's deprecated.

Reply:
I reworked AuthResponse so if fds negotiation fails with an ERROR we just continue without them (like before).